scripts-available/CDB_TableUtils.sql

```diff
@@ -44,6 +45,16 @@ BEGIN
       AND NOT a.attisdropped
       AND a.attrelid = dest_table::oid;
 
+    IF disable_test_quota_per_row
+    THEN
+      -- Disable row-wise quota check trigger for update
+      EXECUTE FORMAT(
```

Comment:
Is this going to run over and over? We are altering the table every run of CDB_TableUtils_ReplaceTableContents?